### PR TITLE
Diffbind: update to v2.10.0 and fix some warnings

### PR DESCRIPTION
**Pull request by mblue9**
This should fix https://github.com/galaxyproject/tools-iuc/issues/2480

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/diffbind/diffbind.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-<tool id="diffbind" name="DiffBind" version="2.6.6.4">
+<tool id="diffbind" name="DiffBind" version="2.10.0">
```
Comment (Slugger70) [inline tools/diffbind/diffbind.xml]: Could you please add a `+galaxy0` as the version of the Galaxy wrapper?

`<tool id="diffbind" name="DiffBind" version="2.10.0+galaxy0">`

Comment (nsoranzo) [reply tools/diffbind/diffbind.xml]: Actually the `+galaxy0` is optional: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions

Comment (mblue9) [reply tools/diffbind/diffbind.xml]: Yep that's what I thought that it's optional. So @Slugger70 is it ok to leave the PR as is?

**Comment by bgruening** [comment]
Thanks all!

**Comment by jasonsaunderswilliams** [comment]
Hi, it seems that this is my issue as kindly established by @mblue9 . I do apologise, but as someone who is not familiar with fixing/updating tools on galaxy. Could someone guide me to where I can teach myself how to implement:
https://github.com/galaxyproject/tools-iuc/pull/2481/commits/2eaa700b2e8d3038d75b7feb50a967563fde9d06

Thanks in advanced.

Jason

**Comment by bgruening** [comment]
@jasonsaunderswilliams I have updated the tool at usegalaxy.eu: https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/bgruening/diffbind/diffbind/2.10.0

Does this help?

**Comment by jasonsaunderswilliams** [comment]
thanks again @bgruening I tried this version and got this following error message:
Fatal error: cannot open file '/opt/galaxy/shed_tools/toolshed.g2.bx.psu.edu/repos/bgruening/diffbind/194e3f2c1d86/diffbind/diffbind.R': No such file or directory

Best wishes,

Jason

**Comment by bgruening** [comment]
my bad, I forget to sync, please try again and if does not work, please send an error report with Galaxy, I will catch then up with you until it works :)

**Comment by jasonsaunderswilliams** [comment]
No worries, I got this error message now. 
[1] "en_US.UTF-8"
2_059E_00OLSwansea_CS001_H3K4me3_hs_i86_trim_bowtie2UPBlkLstRm_MACS_peaks_bed 2_059E_00OLSwansea_CS001_H3K4me3_hs_i86_trim_bowtie2UPBlkLstRm_MACS_peaks_bed  CV1  NA raw
A_ENCFF162UDQ_H3K4Me3_MACS_peaks_bed A_ENCFF162UDQ_H3K4Me3_MACS_pea

I'll send this to galaxy.

Thanks again ;)

**Pull request merged**